app/javascript/components/analytics/widgets/AssessmentsDay.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
+import { PropTypes } from 'prop-types';
```

Comment:
What caused all of the import re-orders, a linter? prettier? manually keeping things consistent?

Reply:
just manually keeping things consistent (https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m) happy to change it back though if this change is not necessary/desired

Reply:
I think we might even be able to automate this with eslint/prettier if we want to in the future: https://medium.com/dooboolab/using-eslint-prettier-and-sort-imports-vscode-extensions-for-formatting-open-source-project-16edf317129d